funcx_endpoint/setup.py

```diff
@@ -48,6 +48,7 @@
     "codecov==2.1.8",
     "pytest-mock==3.2.0",
     "flake8>=3.8",
+    "safety>=1.10.3",
```

Comment:
As an alternative to installing `safety` via the `dev` extra, I think it might be better to make it part of the deps for the relevant tox env.

That way, `safety` can be run without the various test dependencies included.